src/org/parosproxy/paros/extension/option/OptionsConnectionPanel.java

```diff
@@ -508,6 +546,7 @@ private void setProxyChainEnabled(boolean isEnabled) {
 	    spinnerProxyChainPort.setEnabled(isEnabled);
 	    getProxyExcludedDomainsPanel().setComponentEnabled(isEnabled);
 	    chkProxyChainAuth.setEnabled(isEnabled);
+	    chkShowPassword.setEnabled(isEnabled);

```

Comment:
This statement can be removed, the enabled stated is being set in setProxyChainAuthEnabled.